native/packages/react-native-bpk-component-progress/src/styles.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StyleSheet, Platform } from 'react-native';
+import {
+  colorGray100,
+  colorGreen500,
+  colorGreen50,
+  borderRadiusPill,
+  spacingXxl,
+} from 'bpk-tokens/tokens/base.react.native';
+
+export default {
+  Default: StyleSheet.create({
+    trackStyle: {
+      backgroundColor: Platform.select({
+        ios: () => colorGray100,
+        android: () => colorGreen50,
+      })(),
+      borderRadius: borderRadiusPill,
+      width: spacingXxl * 3,
+    },
+    fillStyle: {
+      backgroundColor: colorGreen500,
+      borderRadius: borderRadiusPill,
+    },
+  }),
+  Bar: StyleSheet.create({
```

Comment:
Also when we move them can we avoid multiple invocations of `StyleSheet.create`?